src/server/graphql/models/Invitation/helpers.js

```diff
@@ -41,12 +41,12 @@ export const hashInviteTokenKey = async (uriTokenString) => {
 export const validateInviteTokenKey = async(key, hashStringToCompare) =>
   await compare(key, hashStringToCompare);
 
-export const getInviterInfoAndTeamName = async(teamId, userId) => {
+export const getInviterInfoAndMeetingName = async(meetingId, userId) => {
```

Comment:
refactored the concept of `team` to `meeting` for everything except front end UI stuff.